docs/.link-check-errors.txt

```diff
@@ -1,2 +1,5 @@
 Broken links found by `yarn docs:link-check` that exist:
 
+- https://mui.com/material-ui/customization/right-to-left/
```

Comment:
Are these links normal?

Reply:
They will disappear when we sync monorepo. 😉 
Tested locally. 👌